core/services/keeper/registry_synchronizer_core.go

```diff
@@ -90,8 +90,9 @@ func (rs *RegistrySynchronizer) Start() error {
 				keeper_registry_wrapper.KeeperRegistryUpkeepRegistered{},
 				keeper_registry_wrapper.KeeperRegistryUpkeepPerformed{},
 			},
+			NumConfirmations: 1,
```

Comment:
I think this needs to be the configurable `KEEPER_MINIMUM_REQUIRED_CONFIRMATIONS` should be part of the config passed in from the keeper delegate function.

Reply:
yeah it is already passed as minConfirmations. I was thinking that we would keep the current logic and change this later when we will be removing e.g. keeper head tracking. But yes i can change it now as well

Reply:
yeah it is already passed as minConfirmations. I was thinking that we would keep the current logic and change this later when we will be removing e.g. keeper head tracking. But yes i can change it now as well